File: mathamatics/palindrome_number.py
def palindrome_number_check(num):
    if str(num)== str(num)[::-1]:
        print(str(num)," is a palindrome number")
    else:
        print(str(num)," is not a palindrome number")
palindrome_number_check(6789012)
palindrome_number_check(3456543)

File: mathamatics/palindrome_number.java
class palinbdrome_number
{
    String palindrome_number_check(int num)
    {
        int orginal=num;
        int palin=0;
        int last=0;
        while (num!=0)
           {
               last=num-((num/10)*10);
               palin=palin*10+last;
               num=num/10;
           }
        System.out.print("orginal:"+orginal+"\npalin:"+palin+"\n");
        if (palin==orginal)
            {
               return "Yes";
            }
        else
            {
               return "No";
            }
    }
    public static void main(String[] args)
      {
         
                palinbdrome_number obj =new  palinbdrome_number();     
                System.out.printf("result:"+obj. palindrome_number_check(566765));     

      }
}